packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/FlipbookViewer/FlipbookViewerContainer.js

```diff
@@ -0,0 +1,68 @@
+import React, { useContext } from 'react'
+import asyncStates from '@zooniverse/async-states'
+import PropTypes from 'prop-types'
+import { MobXProviderContext, observer } from 'mobx-react'
+
+import locationValidator from '../../helpers/locationValidator'
+import useSubjectImage from '../SingleImageViewer/hooks/useSubjectImage'
+import FlipbookViewer from './FlipbookViewer'
+
+function useStoreContext() {
+  const store = useContext(MobXProviderContext)?.classifierStore
+  const { frame: defaultFrame } = store.subjectViewer
+  return {
+    defaultFrame
+  }
+}
```

Comment:
The classifier has a `useStores` hook which will do some of this work for you, given a `storeMapper` function. It's in `src/hooks`.